safe-room.lic

```diff
@@ -47,6 +54,23 @@ class SafeRoom
     give_and_take(settings.safe_room_id, settings.safe_room_give, settings.safe_room_take)
   end
 
+  def need_healing(health_hash)
+    settings = get_settings
+    score = 0
+    health_hash.each do |key, array|
+      if key >= 4
```

Comment:
Instead of hardcoding 4, which could change, you could use the `bleeding?` and `checkpoison` helper methods.